tool/lint.sh

```diff
@@ -15,3 +15,17 @@ if [ -n "$bad" ]; then
 	echo "$bad"
 	exit 1
 fi
+
+bad=$(grep -Pzo '\)\s*\n\s*\{' src/main/c/cext/ruby.c || exit 0)
```

Comment:
I changed it to just use a Ruby Regexp. We require ruby >= 2.1 for rubocop anyway.